docs/PT-BR/EXAMPLES.md

```diff
@@ -79,7 +79,7 @@ describe("foo", () => {
   it("bar", () => {
     browser.get("https://example.com");
 
-    const alertElement = element(by.className("el-alert"));
+    const elementoAlerta = element(by.className("elemento-alerta"));
```

Comment:
This part should reflect the same thing as in the English version. And we don't translate the `code.`

Reply:
done